get-zone-name.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Returns the zone name associated with each branch
+
+# Usage
+#   ./get-zone-name.sh develop
+
+# Arguments
+#   - $1 - the branch to check
+
+[ -z "$1" ] && echo "The branch name to check must be provided as the \$1 argument." && exit 1
+
+BRANCH=$1
+
+if  [[ $BRANCH == 'develop' ]] ; then
+  echo "${ZONE_NAME_DEV}"
+elif [[ $BRANCH == 'experimental1' ]] ; then
+  echo "${ZONE_NAME_EXP1}"
+elif [[ $BRANCH == 'experimental2' ]] ; then
+  echo "${ZONE_NAME_EXP2}"
+elif [[ $BRANCH == 'irs' ]] ; then
+  echo "${ZONE_NAME_IRS}"
+elif [[ $BRANCH == 'test' ]] ; then
+  echo "${ZONE_NAME_TEST}"
+elif [[ $BRANCH == 'migration' ]] ; then
+  echo "${ZONE_NAME_MIG}"
+elif [[ $BRANCH == 'staging' ]] ; then
+  echo "${ZONE_NAME_STG}"
+elif [[ $BRANCH == 'master' ]] ; then
+  echo "${ZONE_NAME_PROD}"
+elif [[ $BRANCH == 'dawson' ]] ; then
+  echo "${ZONE_NAME_DAWSON}"
```

Comment:
Pretty sure for us this needs to be 
```
elif [[ $BRANCH == 'prod' ]] ; then
  echo "${ZONE_NAME_PROD}"
```
so, `s/dawson/prod`

Reply:
Updated!